.ci.yaml

```diff
@@ -214,6 +214,40 @@ targets:
       channel: stable
       version_file: flutter_stable.version
 
+  - name: Linux_android custom_package_tests master
```

Comment:
Does the name of these need to start with `Linux_android`? Why not just `Linux custom_package_tests master`?

Reply:
That would use the more generic `Linux` configuration, and then I would have to duplicate all the dependencies in each of the targets. I considered it, but opted for less repetition over preferred naming.

The rigidity of LUCI Cindy naming and how it determines behavior is definitely not one of my favorite parts of LUCI. I would much rather the base config be a property unrelated to name.